src/Routes/SamplePage/SamplePage.js

```diff
@@ -18,48 +14,87 @@ import './sample-page.scss';
-                title: 'Notification title',
-                description: 'notification description'
-            })
+    getSystemsForRos() {
+        fetch(INVENTRY_API_ROOT.concat('/hosts'))
+        .then(res => res.json())
+        .then(
+            result => {
+                this.setState({
+                    systems: result.results
+                });
+            },
+            error => {
+                this.setState({
+                    error
+                });
+            }
         );
-    };
+    }
+
+    renderRow(systems) {
+        if (systems === undefined) {
+            return [];
+        }
 
-    return (
-        <React.Fragment>
-            <PageHeader>
-                <PageHeaderTitle title='Resource Optimization App'/>
-                <p> Instances Details </p>
-            </PageHeader>
-            <Main>
-                <Stack hasGutter>
-                    <StackItem>
-                        <Title headingLevel="h2" size="3xl"> Alerts </Title>
-                        <Button variant='primary' onClick={handleAlert}> Dispatch alert </Button>
-                    </StackItem>
-                    <StackItem>
-                        <SampleComponent/>
-                    </StackItem>
-                    <StackItem>
-                        <Stack hasGutter>
-                            <StackItem>
-                                <Title headingLevel="h2" size="3xl"> Links </Title>
-                            </StackItem>
-                            <StackItem><Link to='/oops'> How to handle 500s in app </Link></StackItem>
-                            <StackItem><Link to='/no-permissions'> How to handle 403s in app </Link></StackItem>
-                        </Stack>
-                    </StackItem>
-                </Stack>
-            </Main>
-        </React.Fragment>
-    );
-};
+        return (
+            systems.map(({ id, display_name: displayName, fqdn, reporter }, index) => (
+                <tr key={index}>
+                    <td>{ id }</td>
+                    <td>{ displayName }</td>
+                    <td>{ fqdn }</td>
+                    <td>70</td>
+                    <td>{ reporter }</td>
+                </tr>
+            ))
+        );
+    }
+
+    render() {
+        const { systems } = this.state;
+
+        return (
+            <React.Fragment>
+                <PageHeader>
+                    <PageHeaderTitle title='Resource Optimization'/>
+                </PageHeader>
+                <Main>
+                    <Card className='pf-t-light  pf-m-opaque-100'>
+                        <Section type='button-group'>
+                            <Button variant='primary' onClick={this.getSystemsForRos} style={ { left: 25, top: 7 } }> Get Systems </Button>
+                        </Section>
+
+                        <CardBody>
+                            <div>
+                                <table className="pf-c-table pf-m-compact">
```

Comment:
Would it be a good idea to use patternfly table component here?
[https://www.patternfly.org/v4/components/table/](https://www.patternfly.org/v4/components/table/)

Reply:
Current implementation is just the basic table rendering to show details.
As you mentioned, I am going to use patternfly+react component for further changes.